testnets/nilliontestnet/chain.json

```diff
@@ -0,0 +1,121 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "nilliontestnet",
+  "status": "live",
+  "network_type": "testnet",
+  "pretty_name": "Nillion Testnet",
+  "chain_id": "nillion-chain-testnet-1",
+  "bech32_prefix": "nillion",
+  "daemon_name": "nilchaind",
+  "node_home": "$HOME/.nillionapp",
+  "key_algos": ["ed25519"],
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "unil",
+        "low_gas_price": 0.0001,
+        "average_gas_price": 0.0001,
+        "high_gas_price": 0.00025
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "unil"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/NillionNetwork/nilchain",
+    "recommended_version": "v0.1.1",
+    "compatible_versions": ["v0.1.1"],
+    "cosmos_sdk_version": "v0.50.6",
+    "consensus": {
+      "type": "cometbft",
+      "version": " v0.38.6"
+    },
+    "cosmwasm_version": "",
+    "cosmwasm_enabled": false,
+    "ibc_go_version": "5.3.2",
+    "genesis": {
+      "genesis_url": "https://raw.githubusercontent.com/NillionNetwork/networks/main/nillion-chain-testnet-1/genesis.json"
+    },
+    "versions": [
+      {
+        "name": "v0.1.1",
+        "recommended_version": "v0.1.1",
+        "compatible_versions": ["v0.1.1"],
+        "cosmos_sdk_version": "v0.50.6",
+        "consensus": {
+          "type": "cometbft",
+          "version": "v0.38.6"
+        },
+        "cosmwasm_version": "",
+        "cosmwasm_enabled": false,
+        "ibc_go_version": "v8.2.0"
+      }
+    ]
+  },
+  "logo_URIs": {
```

Comment:
Please remove logo_URIs

Reply:
Fixed

Reply:
Sorry, not just empty it, but please remove the logo_URIs property entirely

Reply:
Removed logo_URIs